<!-- Launch file for unit testing autonomy. Author : Nicole Maguire -->
<launch>

<!-- time limit (in minutes) should be 10, can be changed for testing purposes -->
<param name="time_limit" value="10"/>
<!-- turn around (in minutes) is how many minutes before the competition ends that the robot should turn around -->
<param name="turn_around" value="1"/>
<param name="location_accuracy" value="0.1"/>
<param name ="optimize_points_turn_around" value="4"/>

</launch>
